systemd-swap.service

```diff
@@ -16,6 +16,7 @@ DeviceAllow=block-blkext r
 DeviceAllow=block-device-mapper r
 DeviceAllow=block-loop rw
 DeviceAllow=block-sd r
+DeviceAllow=block-virtblk rw
```

Comment:
Is write really necessary here? Only read permissions are needed for swap partitions

Reply:
I'm going to test again.

Reply:
No write access actually results in a `swapoff: Not superuser.`, so we seem to need write access.